README.md

```diff
@@ -93,35 +337,36 @@ await consumer.run({
         }
       })
 
-      resolveOffset(message.offset)
+      await resolveOffset(message.offset)
```

Comment:
this comment is actually about the code further up (but Github doesn't let me comment on it).
I think a bit more documentation on the difference between eachMessage and eachBatch would be great.
Also the options in eachBatch such as highWatermark are not clear. I see that this is more a problem of the Kafka docs and not KafkaJS but maybe a link to the right place would be great.

Reply:
done